form/src/main/kotlin/com/ritense/form/domain/FormTaskOpenResultProperties.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.form.domain
+
+import com.fasterxml.jackson.databind.JsonNode
+import java.util.UUID
+
+data class FormTaskOpenResultProperties(
+    val formDefinitionId: UUID,
+    val prefilledForm: JsonNode,
```

Comment:
We might also want to use this class for returning a form for start-events, where prefilling will not be done. Also, a prefilled form is still just a form-definition. 

rename to `formDefinition` ?